net/mwan3/files/usr/sbin/mwan3rtmon

```diff
@@ -107,14 +107,20 @@ mwan3_rtmon_route_handle()
 		local iface=$1
 		tbl=$($IP route list table $tid 2>/dev/null)$'\n'
 
-		if [ "$(cat /var/run/mwan3track/$iface/STATUS)" != "online" ]; then
+		if [ -n "$iface" ] && [ "$(cat /var/run/mwan3track/$iface/STATUS)" != "online" ]; then
```

Comment:
we should not check iface status here, even it is offline we should continue set the route, so remove the check is better.